credentials/jff3/tln-credential.json

```
@@ -0,0 +1,67 @@
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3c-ccg.github.io/vc-ed/plugfest-1-2022/jff-vc-edu-plugfest-1-context.json",
+        "https://w3id.org/security/suites/ed25519-2020/v1",
+	    { "@id": "https://tln.asu.edu/terms/" }
```

This is invalid syntax and the URL doesn't exist.  Based on `tln:courseCareer` later, I'm guessing this should be defining the `tln` prefix?
```suggestion
        { "tln": "https://tln.asu.edu/terms/" }
```